capsicum/Cargo.toml

```diff
@@ -28,10 +28,11 @@ casper = [ "casper-sys", "libnv", "libnv-sys" ]
 [[example]]
 name = "getuid"
 required-features = ["casper"]
+path = "examples/getuid.rs"
```

Comment:
Not entirely sure why the default doesn't work with `cargo publish`, but the following command:

```
cargo publish --dry-run --all-features --keep-going --allow-dirty
```

Fails without this change with the following error message:

```
can't find `getuid` example at `examples/getuid.rs` or `examples/getuid/main.rs`...
```

Reply:
Huh, I wasn't aware of any such requirement, either.  Maybe it's only auto-detect if you don't have an explicit section for the example in Cargo.toml?